typescript/packages/subsurface-viewer/src/layers/wells/wellsLayer.ts

```diff
@@ -384,7 +384,7 @@ export default class WellsLayer extends CompositeLayer<WellsLayerProps> {
             ...this.props.wellLabel,
             id: SubLayerId.LABELS,
             data,
-
+            pickable: true,
```

Comment:
Is this needed if it's already set as default in the `WellLabelLayer`?

Reply:
Couldn't make it work without it.